sections/introduction/MVP.tex

```diff
@@ -0,0 +1,27 @@
+\section{Minimum viable product}
+To be able to evaluate the game with users as fast as possible, we establish a minimal viable product with functional and non-functional requirements.
+
+\subsection{Functional requirements}
+These functional features are essential to make the game work and are the core features of the game.
+
+\begin{itemize}
+    \item Must be possible to start the game.
+    \item Must be able to connect to the server from the clients.
+    \item The users must be able to see their current positions.
+    \item The users must be able to view the playing field, the goals, the players, the ball position and the current score on their mobile device.
+    \item They must be able to win the game.
+    \item The game must be able to be viewed through VR googles.
```

Comment:
```suggestion
    \item The game must be able to be viewed through VR goggles.
```

Reply:
Er det her ikke ret meget en del af "The users must be able to view the playing field, the goals, the players, the ball position, and the current score on their mobile device."